libcob/cconv.c

```diff
@@ -471,3 +471,84 @@ cob_get_collation_by_name (const char *col_name,
 
 	return col_id;
 }
+
+static int
+cob_convert_hex_digit(char h)
+{
+	if (h >= '0' && h <= '9') return h - '0';
+	else if (h >= 'A' && h <= 'F') return 10 + h - 'A';
+	else if (h >= 'a' && h <= 'f') return 10 + h - 'a';
+	else return -1;
+}
+
+static int
+cob_convert_hex_byte(const char *h)
+{
+	int d1 = cob_convert_hex_digit(h[0]);
```

Comment:
minor (general) issue: please add a space before each parameter, so

```diff
 static int
-cob_convert_hex_byte(const char *h)
+cob_convert_hex_byte (const char *h)
 {
-	int d1 = cob_convert_hex_digit(h[0]);
+	int d1 = cob_convert_hex_digit (h[0]);
```

For GC4 there is an indent configuration (unfinished, but "that's to come").